Apps/WebClient/src/ClientApp/src/components/timeline/linearTimeline.vue

```diff
@@ -221,15 +231,20 @@ export default class LinearTimelineComponent extends Vue {
                 :key="dateGroup.key"
                 :ref="dateGroup.key"
             >
-                <component
-                    :is="getComponentForEntry(entry.type)"
+                <div
                     v-for="(entry, index) in dateGroup.entries"
                     :key="entry.type + '-' + entry.id"
-                    :datekey="dateGroup.key"
-                    :entry="entry"
-                    :index="index"
-                    data-testid="timelineCard"
-                />
+                >
+                    <div @click="handleViewEntryDetails(entry)">
```

Comment:
instead of handling it here the event could be handled on the entry itself. That way there is no need for a double div